### ISSUE TITLE
Support dark & light modes for accessibility

### ISSUE THREAD

**Issue by newbold** [neatnik/omg.lol#265]
### Discussed in https://github.com/neatnik/omg.lol/discussions/257

<div type='discussions-op-text'>

<sup>Originally posted by **identithree** October 16, 2022</sup>
There should be a light theme for the official omg.lol pages (homepage, dashboard, info center, etc.) to help out with accessibility. A friend of mine as trouble looking at the site for a while because of their [astigmatism](https://www.vice.com/en/article/ywyqxw/apple-dark-mode-eye-strain-battery-life).</div>

**Comment by newbold** [comment]
First pass of this is complete, with some refinement needed.

**Comment by newbold** [comment]
Closing this since we do technically have a functioning light/dark theme at this point. Will be working on https://lol.design and improving upon all of the accessibility issues that exist currently (contrast in particular).
